pkgs/tmodloader/default.nix

```diff
@@ -0,0 +1,77 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  steam-run,
+  dotnet-runtime_6,
+  dotnet ? dotnet-runtime_6,
```

Comment:
Why not simply use `dotnet` and pass `dotnet = dotnet-runtime_6` in the `callPackage` args?

Reply:
good call, done